packages/public/server/docker-entrypoint-initdb.d/001-init.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.13  Distrib 5.7.28, for Linux (x86_64)
+-- MySQL dump 10.13  Distrib 5.7.26, for Linux (x86_64)
 --
 -- Host: localhost    Database: serlo
 -- ------------------------------------------------------
--- Server version	5.7.28
+-- Server version	5.7.26
```

Comment:
We shoud probably split the fixtures intended for testing from the dump for a couple of reasons:
- We can't update the anonymized dump easily
- We can't run the e2e tests in staging (since they don't have that stuff)
- It's hard to update the test data

There are a couple of things we could do:
- Add an instance for testing purposes via migrations (this could even be in staging & production, too)
- (probably preferable): add another database (with minimal schema / data) that is intended for automated testing, generated via migrations. This way, the anonymized dump can be exchanged / kept up to date. And the testing database can be rolled back whenever (maybe even similar to django after every test), should also be easily applicable to staging / production deployments

Reply:
Follow up issue https://github.com/serlo/serlo.org/issues/293